src/pages/tokens/components/token-edit.vue

```diff
@@ -235,9 +237,52 @@
   </div>
 </template>
 
+<style lang="sass" scoped>
+.terms
+  opacity: 1
+  overflow: hidden
+  max-height: 190px
+  transition-property: max-height, opacity, padding
+  transition-duration: 0.5s
+  .terms-accepted &
+    opacity: 0
+    padding-top: 0px
+    padding-bottom: 0px
+    max-height: 0px
+
+.terms-section
+  transition-duration: 0.5s
+  transition-property: padding
+  .terms-accepted &
+    padding-top: 0px
+
+.form-fields, .footer-buttons
+  opacity: 0
+  overflow: hidden
+  padding-top: 0px
+  padding-bottom: 0px
+  max-height: 0px
+  transition-property: max-height, opacity, padding
+  transition-duration: 0.5s
+  .terms-accepted &
+    opacity: 1
+    max-height: 400px    
+
+@import '~quasar/src/css/variables'
+
+.form-container
+  min-width: auto
+
+@media (min-width: $breakpoint-md-min)
+  .form-container
+    min-width: 900px
+</style>
+
 <script>
 import TokenDetail from "./token-detail.vue";
 import { mapState, mapGetters, mapActions } from "vuex";
+const defaultIcon =
```

Comment:
can we move this into assets or public and import it accordingly?

Reply:
got it!
I included it in the TODO list below.